packages/block-library/src/group/block.json

```diff
@@ -1,18 +1,5 @@
 {
 	"name": "core/group",
 	"category": "layout",
-	"attributes": {
-		"backgroundColor": {
-			"type": "string"
-		},
-		"customBackgroundColor": {
-			"type": "string"
-		},
-		"textColor": {
-			"type": "string"
-		},
-		"customTextColor": {
-			"type": "string"
-		}
-	}
+	"attributes": {}
```

Comment:
Something is inconsistent here. Based on some hooks I guess that backgroundColor and textColor should stay here. It would explain why they aren’t covered in the migration logic for deprecation.

Reply:
They are automatically registered by the hooks.

Reply:
I also think `attributes` isn't mandatory so we can remove this field from the config file and ensure the logic in hooks doesn't break.